### PR TITLE
Review D0500 Slides for activities in day 3

### PR DESCRIPTION
**Pull request by rmarting**
This PR includes the following main changes to add into master repository:

:art: Reviewed TDD Slides
:art: Reviewed CI/CD slides 
:art:  Reviewed User Story Mapping slides

### CODE REVIEW

File: slides/markdown/continuous_integration_continuous_delivery_continuous_deployment.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ waiting for someone to manually test a giant code change.
 * Each change can be **potentially** ready for release to production
 * Builds upon the **continuous integration** practice
 * Consider automating release to a "demo" area
-* Typically implemented with an automation server such as Jenkins
+* Typically implemented with an automation server such as Jenkins or Tekton
```
Comment (jtudelag) [inline slides/markdown/continuous_integration_continuous_delivery_continuous_deployment.md:51]: Tekton is not an Automation Server per se...it is more a framework. We can say a:

"Typically implemented with an automation server such as Jenkins or CI/CD framework such as Tekton"

**Review by jtudelag — COMMENTED** [review]
@rmarting please take a look at my comments, just nitpicking ;).

File: slides/markdown/continuous_integration_continuous_delivery_continuous_deployment.md (hunk 1 above)
Comment (rmarting) [reply slides/markdown/continuous_integration_continuous_delivery_continuous_deployment.md]: It makes sense! Added your suggestion.

**Comment by jtudelag** [comment]
LGTM!

**Comment by rmarting** [comment]
Great! Thanks @jtudelag !

**Pull request merged**